packages/react/src/atomic/Tooltip/CHANGELOG.md

```
@@ -2,6 +2,12 @@
 
 The Tooltip component allows us to display additional information in a non-intrusive way.
 
+## 2025-08-15 `2.6.1`
+
+#### 💡 Others
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)